### PR TITLE
International Address Input pattern page

### LINKED ISSUE #2140: International Address Input Pattern Add to Site
Add pattern documentation to [site patterns](https://www.saltdesignsystem.com/salt/patterns/) section

Design work was done by #1760

### PR DESCRIPTION
**Pull request by lilyvc**
closes #2140 
- Adds the IAI input page
- Adds an overview page to the Patterns section with table linking to Forms and IAI

Index page added
https://saltdesignsystem-git-international-address-pattern-fed-team.vercel.app/salt/patterns/index

IAI page:
https://saltdesignsystem-git-international-address-pattern-fed-team.vercel.app/salt/patterns/international-address-input

### CODE REVIEW

File: site/docs/patterns/index.mdx
Comment (lilyvc) [inline site/docs/patterns/index.mdx:1]: Not sure if there is a design for this page

File: site/docs/patterns/international-address-input.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: International Address Input
+layout: DetailPattern
+sidebar:
+  label: International Address Input
+aliases:
+  - /salt/patterns/international-address-input
```
Comment (lilyvc) [inline site/docs/patterns/international-address-input.mdx:7]: Need resources

File: site/docs/patterns/international-address-input.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  <Diagram
+    src="/img/patterns/international-address-input/toggle_language_2.png"
+    alt="Example showing a form with a show local language toggle on"
+    border
+  />
+</Diagrams>
+
+## Expanding the pattern
+
+If you find a need to expand the pattern, for example to have localized Dynamic templates for more countries or regions, please reach out to the team to discuss.
```
Comment (lilyvc) [inline site/docs/patterns/international-address-input.mdx:147]: need "reach out" link

Comment (origami-z) [reply site/docs/patterns/international-address-input.mdx]: Link in Figma points to support page? https://www.saltdesignsystem.com/salt/support-and-contributions/index

File: site/docs/patterns/index.mdx [hunk 3]
```diff
@@ -1,485 +1,28 @@
...
+    icon={<Image src="/img/patterns/forms.svg" alt="forms"  />}
+    title="Forms"
+    url="./forms"
+    keylineColor="blue"
+    >
+Forms allow you to capture data, gather opinions and receive information in a consistent format.
+  </OverviewCard>
+  
+  <OverviewCard
+    icon={<Image src="/img/patterns/international-address-input.svg" alt="international address input"  />}
```
Comment (origami-z) [inline site/docs/patterns/index.mdx:20]: Is this available?

File: site/docs/patterns/index.mdx [hunk 4]
```diff
@@ -1,485 +1,28 @@
...
-  </Diagram>
-</Diagrams>
-
-<ExampleContainer type="positive">
-  - Core onward actions for the form are grouped together to ensure they remain accessible on browser zoom (e.g., Previous, Next, Cancel, or Submit, Cancel).
-  - Secondary actions which aren’t core to the workflow should be displayed at the opposite end of the button bar.
-  - For a mobile application, buttons can stack (CTA top > Primary > Secondary bottom) to increase button hit area, proximity to thumb, and to utilise mobile scrolling.
-</ExampleContainer>
+   <OverviewCard
+    icon={<Image src="/img/patterns/forms.svg" alt="forms"  />}
```
Comment (origami-z) [inline site/docs/patterns/index.mdx:11]: Is this available?

File: site/docs/patterns/international-address-input.mdx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+The pattern aims to accommodate users worldwide, but your team may be able to take further steps that fall outside the scope of this pattern. For example, check with your tech partners if it is possible to detect user location and automatically adjust the form.
+
+## When to use it
+
+Use the pattern when your user needs to enter an address for a specific country. This could be for shipping, contact, legal or compliance purposes, such as for a client onboarding flow or client detail collection process.
+
+## Pattern content
+
+The pattern consists of Salt Form Field components (for layout guidance, please refer to the [Forms pattern](./forms)). It comprises two formats: Dynamic and Generic.
```
Comment (origami-z) [inline site/docs/patterns/international-address-input.mdx:24]: "Form Field" is a specific component, needs update. 
Something like "various form controls"

File: site/docs/patterns/international-address-input.mdx [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      src: "/img/patterns/international-address-input/select_united_states.png",
+      alt: "Example showing Dynamic template with United States selected",
+    },
+  ]}
+  label="Select United States"
+/>
+
+<Diagrams>
+  <Diagram
+    src="/img/patterns/international-address-input/dynamic_france.png"
```
Comment (origami-z) [inline site/docs/patterns/international-address-input.mdx:50]: These images doesn't feel high res on Mac. Try 2x export in Figma.

File: site/docs/patterns/international-address-input.mdx (hunk 1 above)
Comment (origami-z) [reply site/docs/patterns/international-address-input.mdx:7]: Release without resource link. However, "International Address Input resources" is stretching page horizontally over 100%, need fix

File: site/docs/patterns/international-address-input.mdx (hunk 5 above)
Comment (lilyvc) [reply site/docs/patterns/international-address-input.mdx:24]: asked in the Figma

File: site/docs/patterns/international-address-input.mdx (hunk 6 above)
Comment (lilyvc) [reply site/docs/patterns/international-address-input.mdx:50]: 2x makes them bigger in the site but the same not-high-res quality. These are the same quality as the existing Forms Pattern page, but I can look into other ways to improve this.

File: site/docs/patterns/international-address-input.mdx [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+<Diagram
+  src="/img/patterns/international-address-pattern/customized_address_form.png"
+  alt="Example showing a customized address form"
+  border
+/>
+
+## Toggling the local language
+
+You can add a switch to toggle localization after the country selection to enable users to compare the form with another address or data source.
```
[the remark this answers is not in the document]
Comment (lilyvc) [reply site/docs/patterns/international-address-input.mdx:129]: thank you for reviewing + making suggestions!

File: site/docs/patterns/international-address-input.mdx [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+  <Diagram
+    src="/img/patterns/international-address-input/toggle_language_2.png"
+    alt="Example showing a form with a show local language toggle on"
+    border
+  />
+</Diagrams>
+
+## Expanding the pattern
+
+If you find a need to expand the pattern, for example to have localized Dynamic templates for more countries or regions, please [reach out to the team](/salt/support-and-contributions/index) to discuss.
```
Comment (lilyvc) [inline site/docs/patterns/international-address-input.mdx:146]:
```suggestion
If you need to expand the pattern, for example, to have localized Dynamic templates for more countries or regions, please [reach out to the team](/salt/support-and-contributions/index) to discuss.
```

File: site/docs/patterns/international-address-input.mdx [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+<Diagram
+  src="/img/patterns/international-address-input/generic_format.png"
+  alt="Example showing Generic format"
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+  - Choose the format that best fits the technical capabilities of your app,
+  delivery timescales, and required locations. 
+  - Combine Dynamic format and Generic format if users will input international addresses across the world. You can provide templates using both Generic and Dynamic formats.
```
Comment (lilyvc) [inline site/docs/patterns/international-address-input.mdx:98]:
```suggestion
  - Choose the format that best fits the technical capabilities of your app, delivery timescales and required locations. 
  - Combine Dynamic format and Generic format if users will input international addresses worldwide. You can provide templates using both Generic and Dynamic formats. 
```

**Review by origami-z — DISMISSED** [review]

File: site/docs/patterns/index.mdx [hunk 10]
```diff
@@ -1,485 +1,25 @@
...
 ---
 
-Forms allow you to capture data, gather opinions and receive information in a consistent format.
+Salt is a pattern-driven system. This means we provide a set of simple, high-quality, context-agnostic components and show you how to put them together in different ways, as “patterns”, to solve a specific business problem.
 
-This pattern provides practical and visually enriched guidelines for creating forms that are flexible to business domain, user experience, and channel.
+We recognize the need for UI solutions that are tailored to line of business, product, or persona. Salt Patterns provide you with solutions that you can use “out of the box” or easily modify to cater for nuances in your product experience.
 
-## Form layout
+Each pattern has examples showing how you can put [Salt Components](/salt/components/index) together following the [Salt Foundations](salt/foundations/index). We also offer comprehensive supporting documentation to explain how to use the pattern, how to drive the pattern with business logic, and how to adjust it to suit your business case.
```
Comment (feliciackh) [inline site/docs/patterns/index.mdx:12]: foundations link needs updating

**Review by origami-z — APPROVED** [review]

**Review by JoanaMMoreira — APPROVED** [review]

**Review by feliciackh — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**